css/style.css

```diff
@@ -391,23 +488,2029 @@
+    margin-bottom: 0;
+    font-size: 90px;
+    line-height: 95px;
+  }
+
+  .discription-title {
+    margin: 0;
+    font-weight: medium;
+    line-height: 18px;
+    letter-spacing: 1px;
+    margin-bottom: 65px;
+  }
+
+  .container-image {
+    position: relative;
+    margin: 0;
+    text-align: center;
+    margin-bottom: 20px;
+  }
+
+  .container-image__title {
+    position: absolute;
+    width: 90%;
+  }
+
+  .container-image__img-car {
+    margin-top: 25px;
+    width: 90%;
+    height: auto;
+  }
+
+  .special-list {
+    display: flex;
+    flex-wrap: wrap;
+    max-width: 570px;
+    list-style-type: none;
+    margin-bottom: 60px;
+  }
+
+  .special-list__item:nth-child(1) {
+    padding-left: 85px;
+    margin-right: 80px;
+    background-image: url(/img/Speed_ipad.svg);
+    background-repeat: no-repeat;
+    background-position: left;
+    background-position-y: 15px;
+  }
+
+  .special-list__item:nth-child(2) {
+    padding-left: 60px;
+    background-image: url(/img/Road\ icon_ipad.svg);
+    background-repeat: no-repeat;
+    background-position: left;
+    background-position-y: 15px;
+    margin-bottom: 65px;
+  }
+
+  .special-list__item:nth-child(3) {
+    padding-left: 50px;
+    padding-top: 2px;
+    margin-right: 45px;
+    margin-left: 35px;
+    background-image: url(/img/steering-wheel_ipad.svg);
+    background-repeat: no-repeat;
+    background-position: left;
+  }
+
+  .special-list-text {
+    margin: 0;
+    font-size: 50px;
+    line-height: 57px;
+    font-weight: bold;
+  }
+
+  .special-list-description {
+    font-size: 12px;
+    line-height: 15px;
+    font-weight: lighter;
+  }
+
+  .special-list-text__description {
+    font-size: 20px;
+    font-weight: medium;
+    line-height: 24px;
+  }
+
+  .special-list-item__text {
+    font-size: 16px;
+    font-weight: bold;
+    line-height: 20px;
+    color: #e51937;
+    text-transform: uppercase;
+  }
+
+  .order-button-red {
+    background-color: #e51937;
+    color: #fff;
+  }
+
+  .arrow-downd {
+    transition: 1s;
+  }
+
+  .arrow-downd:hover {
+    transform: scale(1.2);
+  }
+
+  .future-drive {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .future-drive__title {
+    position: absolute;
+    top: 1010px;
+    left: 430px;
+    white-space: nowrap;
+    font-size: 70px;
+    line-height: 85px;
+    color: #e51937;
+  }
+
+  .future-drive-img {
+    margin: 0;
+    font-size: 0;
+  }
+
+  .future-drive__img {
+    width: 100%;
+  }
+
+  .future-drive__img-mobile {
+    display: none;
+  }
+
+  .future-drive__img-big {
+    display: none;
+  }
+
+  .future {
+    background-color: #e51937;
+    color: #fff;
+  }
+
+  .container-future {
+    display: flex;
+    flex-wrap: wrap;
+    padding-top: 30px;
+  }
+
+  .container-text {
+    margin-right: 100px;
+    padding-left: 35px;
+  }
+
+  .container-text__tittle {
+    margin-bottom: 22px;
+    font-size: 30px;
+    line-height: 37px;
+    letter-spacing: 1.4px;
+    }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6183733)